src/components/Personalization/helper/click.js

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { uuid } from "../../../utils";
+import { awaitSelector } from "../../../utils/dom";
+import ALLOY_KEY from "../../../constants/clickTrackingAttribute";
+import { selectNodesWithEq, setAttribute } from "./dom";
+
+export default (settings, store) => {
+  const key = uuid();
```

Comment:
This isn't a big deal, but it might be good to just move this line down inside the `.then` so that we only generate a UUID if elements matching the selector are found.

Reply:
Done.